modyn/config/schema/pipeline/evaluation/metric.py

```diff
@@ -27,10 +27,15 @@ def validate_evaluation_transformer_function(cls, value: str) -> str | None:
             raise ValueError("Function 'evaluation_transformer_function' could not be parsed!") from exc
         return value
 
+    @field_validator("evaluation_transformer_function", mode="before")
+    @classmethod
+    def clean_evaluation_transformer_function(cls, value: str | None) -> str:
+        return value or ""
```

Comment:
since the metric refactor PR also touches this, should be sure to sync this. What do you mean by clean btw? That it cannot be None?

Reply:
Not sure what my intention was here, I'll try to rebase it after merging the other PR.